src/ts/background/store/settings/actions.ts

```diff
@@ -1,8 +1,9 @@
 import { Action } from 'redux';
 
-export type ThemeActionTypes = 'DARK_THEME' | 'LIGHT_THEME';
-
-export type SettingsActions = Action<ThemeActionTypes>;
+export type ThemeActionTypes = 'DARK_THEME' | 'LIGHT_THEME' | 'RETURN_TO_HOME' | 'SET_FEATURE_ID';
```

Comment:
um, should this be `SettingsActionTypes` (and should we get rid of the other theme stuff?)

Reply:
Sure.
Dark mode would be cool, I intentionally left that, even such simple thing like `filter: invert(1);` looks ok. There must be additional view to import/export config, it can also contain switch for theme.

But - are we going the way I proposed? If you have nothing fundamentally different and better in mind and want to use what's in this branch, then I'll get back to it (during the weekend) and work on it further.

Reply:
Sure.
Dark mode would be cool, I intentionally left that, even such simple thing like filter: invert(1); looks ok. There must be additional view to import/export config, it can also contain switch for theme.

But - are we going the way I proposed? If you have nothing fundamentally different and better in mind and want to use what's in this branch, then I'll get back to it (during the weekend) and work on it further.

Reply:
the type should be still renamed though, even if we're keeping the dark mode :)

for the general point - I say yes. I'm sure I'll have a bunch more comments on details, but general approach makes sense to me (and it'd be nice to make it work without having to reload, and maybe have feature auto-discovery too/etc, but all these things are incremental improvements we can work on :) )

Reply:
I had it working without reloading in `Settings foundations` commit but then extracted shortcuts from other settings in last minute and something broke, but that's definitely something that will work